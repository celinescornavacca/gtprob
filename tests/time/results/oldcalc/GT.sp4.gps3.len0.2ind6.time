0.25user 0.09system 0:00.47elapsed 74%CPU (0avgtext+0avgdata 29476maxresident)k
124inputs+0outputs (0major+12546minor)pagefaults 0swaps
