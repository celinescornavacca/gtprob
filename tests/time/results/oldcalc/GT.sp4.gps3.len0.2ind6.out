-16.4345719279
